sentry/src/main/java/io/sentry/Hub.java

```diff
@@ -339,6 +339,10 @@ public void close() {
             ((Closeable) integration).close();
           }
         }
+
+        withScope(scope -> scope.clear());
```

Comment:
late to the party, but this is not gonna do much, because `withScope` will duplicate the scope, and then we're trying to clear the duplicated scope, but the main scope stays untouched. This should be changed to `configureScope`. I think we need a custom lint rule to never use withScope on mobile 😅 this is the second time we introduced something like this (this one is not as severe though).